docs/auditors/mounts.md

```diff
@@ -0,0 +1,273 @@
+| -s      | --paths   | List of sensitive paths that shouldn't be mounted.                   | [default sensitive host paths list](#Default-sensitive-host-paths-list)  |
+
+Also see [Global Flags](/README.md#global-flags)
+
+#### Default sensitive host paths list
+
+| Host path                 | Description                                                                              |
+| :------------------------ | :--------------------------------------------------------------------------------------- |
+| /proc                     |  Pseudo-filesystem which provides an interface to kernel data structures                                                  |
+| /var/run/docker.sock      |  Unix socket used to communicate with Docker daemon                                   |
+| /                         |  Filesystem's root |
+| /etc                      |  Directory that usually contains all system related configurations files         |
+| /root                     |  Home directory of the `root` user                                             |
+| /var/run/crio/crio.sock   |  Unix socket used to communicate with the CRI-O Container Engine                                                 |
+| /home/admin               |  Home directory of the `admin` user        |
+| /var/lib/kubelet          |  Directory for Kublet-related configuration                                                             |
+| /var/lib/kubelet/pki      |  Directory containing the certificate and private key of the Kublet                                                               |
+| /etc/kubernetes           |  Directory containing Kubernetes related configuration              |
+| /etc/kubernetes/manifests |  Directory containing manifest of Kubernetes components                                                         |
+
+## Examples
+
+```
+$ kubeaudit mounts -f auditors/mounts/fixtures/proc-mounted.yml
+
+---------------- Results for ---------------
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: pod
+    namespace: proc-mounted
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: proc-volume (/proc -> /host/proc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      MountName: proc-volume
+      MountPath: /host/proc
+      MountReadOnly: false
+      MountVolume: proc-volume
+      MountVolumeHostPath: /proc
+
+```
+
+### Example with Config File
+
+A custom paths list can be provided in the config file. See [docs](docs/all.md) for more information. These are the host
+paths you'd like to have kubeaudit raise an error when they are mounted in a container.
+
+`config.yaml`
+
+```yaml
+---
+auditors:
+  mounts:
+    paths: ["/etc", "/var/run/docker.sock"]
+```
```

Comment:
When I tested locally, I also had to turn this auditor on by setting `mounts`to true, like so:

```yaml
enabledAuditors:
  mounts: true
 ```

Reply:
Another suggestion, to make it more explicit (completely optional). Instead of paths, how about `pathsToWatch`?

Reply:
I went for the simple `paths` to be more aligned with other auditors that also use simple key (`add`, `image`, `cpu`)  that are also the same as the CLI options. But we can totally switch to `pathsToWatch` if you find it better.

Reply:
I see. For the capabilities auditor, however, we add capabilities to an allowAddList  so `kubeaudit` won't raise errors for those.  So it made me think that paths would have a similar effect - listed paths would be the ones `kubeaudit` would ignore. However it's the opposite - `kubeaudit` will only watch for those specific paths, everything else will be ignored. That's why I thought of a more explicit name... what do you think?

Reply:
Ok understood. I find the phrasing a bit weird however :) I was about to suggest `sensitivesPaths` but I guess the issue is the same, it may not be quite clear that it work as a deny-list and not as an allow-list. So if you prefer, yes we can go for `pathsToWatch`

Reply:
yeah... let's leave it like then, then. And hopefully the documentation will make it clear! 😁

Reply:
or maybe `denyPathsList`? :P

Reply:
@jcbbc can you please change it to `denyPathsList`? with that, we're good to go! thanks! 🙏

Reply:
Sorry, I missed your previous comment. I've made the change.

Reply:
Thanks a lot, @jcbbc!! :)